bokehjs/test/core/util/templating.coffee

```diff
@@ -0,0 +1,93 @@
+{expect} = require "chai"
```

Comment:
thanks for finally getting this added. we've handle multiple regressions because we never tested it.